src/sys/linux/os.rs

```diff
@@ -0,0 +1,9 @@
+pub fn errno() -> i32 {
+    panic!("no C-compatible errno variable");
```

Comment:
this is fine for now but could you open an issue to discuss what would be fate of `last_os_error`? I think redox returns 0 (or some other value) here instead of panicking.